src/ucp/core/ucp_context.c

```diff
@@ -93,6 +93,11 @@ static ucs_config_field_t ucp_config_table[] = {
    "is zero or negative",
    ucs_offsetof(ucp_config_t, ctx.rndv_thresh_fallback), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"USE_RNDV_PUT_ZCOPY", "n", "Use put_zcopy in RNDV protocol.\n"
+   "n - Do not use put_zcopy in RNDV protocol, use default get_zcopy"
+   "y - Use put_zcopy in RNDV protocol",
+   ucs_offsetof(ucp_config_t, ctx.use_rndv_put_zcopy), UCS_CONFIG_TYPE_BOOL},
```

Comment:
I think context is too global place for protocol switching. Maybe better to make the decision per EP?